File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* my_string) {
    char* answer = (char*)malloc(sizeof(char)* strlen(my_string));
    memset(answer,0,sizeof(char)*strlen(my_string));
    for(int i=0;i<=strlen(my_string);i++)
    {
        answer[i] = my_string[strlen(my_string) - i - 1];
        printf("%c %c \n",answer[i],my_string[strlen(my_string) - i]);
    }

    return answer;
}

File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string) {
        // StringBuilder를 사용하여 문자열을 반전
        StringBuilder str = new StringBuilder(my_string);
        System.out.println(str.reverse());  // 반전된 문자열 출력
        
        return str.toString();  // 반전된 문자열을 반환
    }
}

